Makefile

```
@@ -193,3 +193,34 @@ _OS:=	${_OS:C/-.*//}
 .if "${_OS}" != "${OS}"
 .error Expected OS version ${OS} for ${_CONFIGDIR}; to continue anyway set OS=${_OS}
 .endif
+
+
+# Rules for establishing OPNsense repos as the package source.   This is
+# to fix the pkg versioning issues with upstream.  This fixes the build
+# machine itself.
+
+/usr/local/bin/git:
+	pkg install -y git
+
+/root/pkg-static:
+	cp /usr/local/sbin/pkg-static /root/pkg-static
+
+fix: /usr/local/bin/git update /root/pkg-static
+	/root/pkg-static info | \
+		awk '{print $$1}' | \
+		xargs /root/pkg-static delete -fy
+	cp -a ${COREDIR}/src/etc/pkg /usr/local/etc/
```

According to the script, the `pkg` directory should still exist at the destination, right? Then the last slash is too much:

```suggestion
	cp -a ${COREDIR}/src/etc/pkg /usr/local/etc
```